packages/components/psammead-media-player/src/Guidance/index.jsx

```diff
@@ -61,15 +61,15 @@ const GuidanceMessage = styled.span`
 
 const IconWrapper = styled.div``;
 
-const Guidance = ({ message, service, children }) => {
+const Guidance = ({ guidanceMessage, service, children }) => {
   return (
     <StyledGuidance service={service}>
```

Comment:
This alignment issue will likely get picked up in the UX review:

![Screenshot 2019-10-31 at 12 20 58](https://user-images.githubusercontent.com/1430787/67946431-512eb000-fbd9-11e9-9cca-e8a026cc7fd6.png)